linetools/guis/line_widgets.py

```diff
@@ -0,0 +1,298 @@
+""" Module for line widets
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+from PyQt4 import QtGui
+from PyQt4 import QtCore
+
+import numpy as np
+
+from astropy.table import Table
+
+from linetools.guis import utils as ltgu
+from linetools.lists.linelist import LineList
+
+# #####
+class PlotLinesWidget(QtGui.QWidget):
+    """ Widget to set up spectral lines for plotting
+    """
+    def __init__(self, parent=None, status=None, init_llist=None, init_z=None):
+        """
+        Parameters
+        ----------
+        parent
+        status
+        init_llist
+        init_z
+
+        Returns
+        -------
+
+        """
+        super(PlotLinesWidget, self).__init__(parent)
+
+        # Initialize
+        if not status is None:
+            self.statusBar = status
+        if init_z is None:
+            init_z = 0.
+
+        # Create a dialog window for redshift
+        z_label = QtGui.QLabel('z=')
+        self.zbox = QtGui.QLineEdit()
+        self.zbox.z_frmt = '{:.7f}'
+        self.zbox.setText(self.zbox.z_frmt.format(init_z))
+        self.zbox.setMinimumWidth(50)
+        self.connect(self.zbox, QtCore.SIGNAL('editingFinished ()'), self.setz)
+
+        # Create the line list
+        self.lists = ['None', 'ISM', 'Strong', 'Galaxy', 'H2', 'EUV', 'OVI']
```

Comment:
Not convinced by having a 'OVI' linelist. Maybe worth considering having a 'Custom' linelist that can be given as a parameter in the command line (?)